src/test/ml-modules/root/test/suites/matching/allow-match.xqy

```diff
@@ -0,0 +1,31 @@
+xquery version "1.0-ml";
+
+import module namespace matcher = "http://marklogic.com/smart-mastering/matcher"
+at "/ext/com.marklogic.smart-mastering/matcher.xqy";
+
+import module namespace test = "http://marklogic.com/roxy/test-helper" at "/test/test-helper.xqy";
+
+(: Force update mode :)
```

Comment:
can you add disable function mapping to these? just for fun?

Reply:
oversight; will do